.travis.yml

```diff
@@ -7,20 +7,20 @@ env:
   global:
     - GETH_BASE_INSTALL_PATH=$TRAVIS_BUILD_DIR
   matrix:
-    - TOX_ENV=py27-core-stdlib
-    - TOX_ENV=py34-core-stdlib
-    - TOX_ENV=py35-core-stdlib
-    - TOX_ENV=py27-core-gevent
-    - TOX_ENV=py34-core-gevent
-    - TOX_ENV=py35-core-gevent
-    - TOX_ENV=py35-integration-ethtestrpc
-    - TOX_ENV=py35-integration-goethereum GETH_VERSION=v1.6.0
-    - TOX_ENV=py35-integration-goethereum GETH_VERSION=v1.6.1
-    - TOX_ENV=py35-integration-goethereum GETH_VERSION=v1.6.4
-    - TOX_ENV=py35-integration-goethereum GETH_VERSION=v1.6.5
-    - TOX_ENV=py35-integration-goethereum GETH_VERSION=v1.6.6
-    - TOX_ENV=py35-integration-goethereum GETH_VERSION=v1.6.7
-    - TOX_ENV=flake8
+    - TOX_POSARGS="-e py27-core-stdlib"
+    - TOX_POSARGS="-e py34-core-stdlib"
+    - TOX_POSARGS="-e py35-core-stdlib"
+    - TOX_POSARGS="-e py27-core-gevent"
+    - TOX_POSARGS="-e py34-core-gevent"
+    - TOX_POSARGS="-e py35-core-gevent"
+    - TOX_POSARGS="-e py35-integration-ethtestrpc -e py34-integration-ethtestrpc -e py35-integration-ethtestrpc"
```

Comment:
Looks like the first arg here should be `py27-integration-ethtestrpc`

Reply:
Good catch.